src/main/java/lt/vu/trip/service/trip/TripServiceImpl.java

```diff
@@ -0,0 +1,24 @@
+package lt.vu.trip.service.trip;
+
+import lt.vu.trip.entity.Trip;
+import lt.vu.trip.repository.TripRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+
+@Component
+public class TripServiceImpl implements TripService {
+
+	@Autowired
+	private TripRepository repo;
+
+	public List<Trip> getAll() {
+		return this.repo.findAll();
+	}
+
+	public boolean createNew(Trip trip) {
+		this.repo.saveAndFlush(trip);
```

Comment:
this tokiame kontekste nereikia, nes su niekuo kitu name nesikerta. Tas pats visur kitur

Reply:
Pašalinta.